DUI3-DX/Converters/Revit/Speckle.Converters.RevitShared/Helpers/ParameterValueExtractor.cs

```diff
@@ -164,7 +164,7 @@ public IRevitElementId GetValueAsElementId(IRevitElement element, RevitBuiltInPa
   {
     if (!TryGetValueAsElementId(element, builtInParameter, out var elementId))
     {
-      throw new SpeckleConversionException();
```

Comment:
Was this the original code? or a change you made when doing the testing changes?